xwiki-platform-core/xwiki-platform-web/xwiki-platform-web-war/src/main/webapp/resources/js/xwiki/usersandgroups/usersandgroups.js

```diff
@@ -57,44 +57,42 @@ window.MSCheckbox = Class.create({
     this.state = defaultState;
     this.states = [0,1,2]; // 0 = undefined; 1 = allow, 2 = deny
     this.nrstates = this.states.length;
-    this.images = [
-      "$xwiki.getSkinFile('js/xwiki/usersandgroups/img/none.png')",
-      "$xwiki.getSkinFile('js/xwiki/usersandgroups/img/allow.png')",
-      "$xwiki.getSkinFile('js/xwiki/usersandgroups/img/deny1.png')"
+    this.icons = [
+      "",
+      "$escapetool.javascript($services.icon.renderHTML('check'))",
+      "$escapetool.javascript($services.icon.renderHTML('cross'))"
```

Comment:
Would be cleaner to move out the velocity calls, as it's recommended in https://dev.xwiki.org/xwiki/bin/view/Community/DevelopmentPractices#HJavaScriptBestPractices (and what you used in https://github.com/xwiki/xwiki-platform/pull/2648/files#diff-14e05ae329b087de0d0f45b460cdb1b33da4382f3d9c806125daf81cfac152bbR51 for example)

Reply:
Addressed in https://github.com/xwiki/xwiki-platform/pull/2649/commits/78ca48e9fb3ed17d074ec0d0aefd8f615db47672 :+1: 
I tested this change manually and as far as I could see it did not break anything (at least I fixed mistakes until I could notice it working again :) ).